test/e2e/v1beta1/e2e_test.go

```diff
@@ -706,4 +708,54 @@ var _ = Describe("For a Kubernetes cluster with Tekton and build installed", fun
 		})
 	})
 
+	Context("when tolerations are specified", func() {
```

Comment:
Because of node taints

```suggestion
	Context("when tolerations are specified", Serial, func() {
```